### PR TITLE
media-gfx/gnofract4d: Version bump 3.14.1

### PR DESCRIPTION
**Pull request by cjmayo**
- Update to EAPI 6
- Moved from SourceForge to GitHub
- Remove gconf dependency fixing:
    Gentoo-Bug: https://bugs.gentoo.org/384205
- Add slot operators for libpng and jpeg
- Fix desktop file validation problems
- Make manual readable by yelp

----

Project moved as noted at:
https://sourceforge.net/projects/gnofract4d/

May also fix :
https://bugs.gentoo.org/show_bug.cgi?id=564298

Bundled manual is not readable by yelp because it references the file commands.xml that is generated by createdocs.py, but it needs to be able to find the binary fract4dc module to do this. I couldn't get this to work and therefore just patched out commands.xml.
Yelp could be a RDEPEND - but I suggest would need a USE flag, if you aren't running GNOME it drags in a lot of other packages.

### CODE REVIEW

File: media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+HOMEPAGE="http://edyoung.github.io/gnofract4d/"
+SRC_URI="https://github.com/edyoung/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="x11-libs/gtk+:2
+	>=media-libs/libpng-1.4:=
```
Comment (SoapGentoo) [inline media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild]: just make this `media-libs/libpng:0=`

File: media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	virtual/jpeg:=
```
Comment (SoapGentoo) [inline media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild]: `virtual/jpeg:0`

File: media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	"${FILESDIR}"/gnofract4d-3.14-manual.patch
+)
+
+S="${WORKDIR}"/${PN}-${MY_PV}
+
+python_compile_all() {
+	distutils-r1_python_compile_all
+
+	# Needs fixing to be able to generate commands.xml
+	"${PYTHON}" createdocs.py || die
```
Comment (SoapGentoo) [inline media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild]: Try `EPYTHON` before using `PYTHON`

File: media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+python_install_all() {
+	distutils-r1_python_install_all
+	rm -rf "${D}"/usr/share/doc/${PN} || die
```
Comment (SoapGentoo) [inline media-gfx/gnofract4d/gnofract4d-3.14.1.ebuild]: `rm -rf "${ED%/}"/usr/share/doc/${PN} || die`

**Comment by cjmayo** [comment]
Thanks for the review, all implemented.
Plus removed call to non-existent distutils-r1_python_compile_all in  python_compile_all().

**Pull request closed without merging**